src/types/Module.interface.ts

```diff
@@ -48,3 +51,7 @@ export interface Web3FunctionParams {
   web3FunctionArgsHex: string | null;
   web3FunctionSchema?: Web3FunctionSchema;
 }
+
+export interface TriggerParams {
+  triggerConfig: TriggerConfig | null;
```

Comment:
same comment about `Config`
```suggestion
  trigger: TriggerConfig | null;
```

Reply:
fixed 0bf10e5123ee4affee7084cbd7af3d7a16c62f6f